starter-code/src/viking.js

```diff
@@ -1,11 +1,71 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength = strength;
+
+  Soldier.prototype.attack = function() {
+    return this.strength;
+  };
+
+  Soldier.prototype.receiveDamage = function(damage) {
+    this.health -= damage;
+  };
+}
 
 // Viking
-function Viking() {}
+function Viking(name, health, strength) {
+  this.name = name;
+
+  Soldier.call(this, health, strength);
+
+  this.receiveDamage = function(damage) {
```

Comment:
remember to add this function to the prototype of Viking instead just after line 34